handlers/lib/config.js

```diff
@@ -61,12 +62,23 @@ const initEnvVars = ftpActionType => {
   }
 };
 
-const getEnv = varName => {
+const getEnv = (varName, altVars = []) => {
```

Comment:
`altVars` here specifies a list of alternative environment variables that can be set in lieu of `varName`.

I believe it's technically possible to set up a non-authenticated SFTP endpoint, in which case this code isn't needed, but support for non-authenticated SFTP will come in exchange of slightly worse error messages for typical use cases.